unity/Holo/Assets/Scripts/ModelWithPlate.cs

```diff
@@ -162,25 +195,29 @@ private void LoadInstance(string newInstancePath, bool newIsPreview)
             // We should settle whether the stored object has or has not BlendShapeAnimation (I vote not,
             // but any decision is fine, just stick to it).
             // After new import STL->GameObject is ready.
-            Debug.LogWarning("BlendShapeAnimation component not found inside " + newInstancePath + ", adding");
+            Debug.LogWarning("BlendShapeAnimation component not found inside " + newInstanceIndex + ", adding");
             instanceAnimation = instance.AddComponent<BlendShapeAnimation>();
         }
         // default speed to play in 1 second
         int blendShapesCount = skinnedMesh.sharedMesh.blendShapeCount;
         if (blendShapesCount != 0) {
             instanceAnimation.Speed = skinnedMesh.sharedMesh.blendShapeCount;
         } else {
-            Debug.LogWarning("Model has no blend shapes " + newInstancePath);
+            Debug.LogWarning("Model has no blend shapes " + newInstanceIndex);
         }
         instanceAnimation.Playing = true;
         // TODO: this should be changed into "throw new...", not a warning.
         // After new import STL->GameObject is ready.
         Animator animator = instance.GetComponent<Animator>();
         if (animator != null) {
-            Debug.LogWarning("Animator component found but not wanted inside " + newInstancePath + ", removing");
+            Debug.LogWarning("Animator component found but not wanted inside " + newInstanceIndex + ", removing");
             Destroy(animator);
         }
 
+        // Assign material, designating preview / not preview
+        // TODO: in the actual application, the "preview" flag should load a simpler mesh
+        skinnedMesh.sharedMaterial = newIsPreview ? MaterialPreview : MaterialNonPreview;

```

Comment:
After loading `MaterialNonPreview` is set to some invalid clipping plane (or the one connected to collection plate material) which cause rendering not visible (fully clipped) model

Reply:
This means a wrong material is attached in the scene. This needs fixing in the scene, which was already modified in other PRs, so I'd rather fix it later on master :)

Reply:
I looked into it -- actually DefaultMaterial is broken (it has "clipping plane" shader instead of "standard"), moreover materials are incorrectly assigned in the scene on ModelWithPlate MonoBehaviour -- DefafaulMaterialPreview was assigned to MaterialNonPreview, and DefaultMaterial to MaterialPreview :)

I committed a fix to DefaultMaterial. The fix on scene will be done after merging.